lib/ttfunk/table/name.rb

```diff
@@ -45,10 +45,30 @@ def strip_extended
       attr_reader :compatible_full
       attr_reader :sample_text
 
+      COPYRIGHT_NAME_ID = 0
+      FONT_FAMILY_NAME_ID = 1
+      FONT_SUBFAMILY_NAME_ID = 2
+      UNIQUE_SUBFAMILY_NAME_ID = 3
+      FONT_NAME_NAME_ID = 4
+      VERSION_NAME_ID = 5
+      POSTSCRIPT_NAME_NAME_ID = 6
+      TRADEMARK_NAME_ID = 7
+      MANUFACTURER_NAME_ID = 8
+      DESIGNER_NAME_ID = 9
+      DESCRIPTION_NAME_ID = 10
+      VENDOR_URL_NAME_ID = 11
+      DESIGNER_URL_NAME_ID = 12
+      LICENSE_NAME_ID = 13
+      LICENSE_URL_NAME_ID = 14
+      PREFERRED_FAMILY_NAME_ID = 16
+      PREFERRED_SUBFAMILY_NAME_ID = 17
+      COMPATIBLE_FULL_NAME_ID = 18
+      SAMPLE_TEXT_NAME_ID = 19
+
       def self.encode(names, key = '')
         tag = Digest::SHA1.hexdigest(key)[0, 6]
 
-        postscript_name = Name::String.new(
+        postscript_name = Name::NameString.new(
```

Comment:
There's no need to namespace this any more, is there?

Reply:
Nope :)